tst/EnergyPlus/unit/AirflowNetworkBalanceManager.unit.cc

```diff
@@ -216,4 +216,127 @@ namespace EnergyPlus {
 		SurfaceWindow.deallocate();
 
 	}
+
+	TEST_F( EnergyPlusFixture, AirflowNetworkBalanceManager_TestWarningMessage ) {
```

Comment:
@lgu1234 I would make the test name more specific, perhaps "AirflowNetworkBalanceManager_TestTriangulerWindowWarning"